tests/baselines/reference/duplicatePackage.types

```diff
@@ -1,22 +1,22 @@
 === /src/a.ts ===
 import { a } from "a";
->a : (x: X) => void
+>a : (x: default) => void

```

Comment:
The full type here would be `module("x").default`, but there's no handle to `module("x")` available in scope. (And I added `defaultDeclarationEmitDefaultImport.ts` as a test to prove that all is named as expected when there's a viable alias in scope.) When there's an error like that, our current behavior when ignore errors is set (as is the case here) is simply to print what we can - namely the name of the symbol.

Effectively, the new behavior is to consider the name of a `default` symbol as `default` unless an override is passed (not the case in type baseline emit), or we're at the very LHS of a name in the same file as the `default` symbol's declaration.